Parse/cloud/main.js

```diff
@@ -100,6 +168,44 @@ Parse.Cloud.define("fetchUserCellData", async (request) => {
   return results;
```

Comment:
for some reason git won't let me comment above but shouldn't we get rid of `existsFollow`, that's not being used anymore by the client right?